src/Compilers/VisualBasic/Portable/Emit/NoPia/EmbeddedTypesManager.vb

```diff
@@ -194,6 +194,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit.NoPia
                     TypeKind.Enum,
                     TypeKind.Delegate
 
+                    If type.IsTupleType Then
+                        type = type.TupleUnderlyingType
```

Comment:
>type = type.TupleUnderlyingType [](start = 24, length = 31)

This probably was fine when we were simply propagating TypeKind of underlying type through TupleTypeSymbol. I think we are not doing this any more. Consider moving this substitution out of the Select Case. This also applies to C# code.